example/src/index.scss

```diff
@@ -76,3 +76,17 @@ label {
 
 }
 
+
+/*Style Changes Error / Touched */
+
+.div{
```

Comment:
Deze file is nu een SCSS file. Ik stel voor dat je dit even refactored zodat elke "pagina" een eigen file krijgt die je vervolgens binnentrekt in deze file (```@import '...'```)

bv.
```
.buttons {
  .wrapper{}
  .flexbox {}
}
```

Reply:
Dus een aparte scss file voor buttons enz?